src/main/telemetry/crsf.c

```diff
@@ -372,7 +373,8 @@ void crsfFrameHeartbeat(sbuf_t *dst)
 int8_t    destination_add ( Destination Device address )
 int8_t    origin_add ( Origin Device address )
 */
-void crsfFramePing(sbuf_t *dst)
+
+MAYBE_UNUSED static void crsfFramePing(sbuf_t *dst)
```

Comment:
Perhaps should be within `#if defined(USE_CRSF_CMS_TELEMETRY)` tag?

Reply:
Its `defined(USE_CRSF_V3) && defined(USE_CRSF_CMS_TELEMETRY)`.  It is easy to fix here, but there will be some code with overcomplicated condition. So it will either become ugly (duplicating condition) or inconsistent (MAYBE_UNUSED in nontrivial cases only). 

I really don't know.  I can update easy cases if you think that it helps.